backend/src/auth/utils.py

```diff
@@ -117,11 +126,16 @@ def encode_jwt_token(username: str, user_id: uuid.UUID) -> dict:
     return data
 
 
-def get_user_id_from_request(request: Request) -> Optional[uuid.UUID]:
+def get_token_from_request(request: Request) -> Optional[str]:
     authorization = request.headers.get("Authorization")
-    if not authorization:
-        return None
-    auth_token = authorization.split(" ")[1]
+    scheme, token = get_authorization_scheme_param(authorization)
+    if not authorization or scheme.lower() != "bearer":
+        raise NotAuthenticated
+    return token
+
+
+# TODO to change with verify_user_id you can Depends user_id = Annotate(UUID, Depends(verify_user_id))
```

Comment:
Why you don't create issue for this??

Reply:
because I'm not a professional programmer yet and I don't know how to work in a team